### ISSUE TITLE
BootstrapV4

### ISSUE THREAD

**Issue by mano87** [RalfEggert/travello-view-helper#1]
Is there already a version that is compatible with Bootstrap Version 4? :-)

**Comment by RalfEggert** [comment]
No, and there is nothing planned so far. I would recommend to fork the library and update the templates.
